src/main/java/com/example/sinitto/guardGuideline/controller/GuardGuidelineController.java

```diff
@@ -56,4 +56,12 @@ public ResponseEntity<GuardGuidelineResponse> getGuardGuideline(@RequestParam("c
         return ResponseEntity.ok(guardGuidelineService.readGuardGuideline(callbackId, guidelineId));
     }
 
+    @Operation(summary = "특정 가이드라인 삭제", description = "보호자용 API입니다.")
+    @GetMapping("/delete")
+    public ResponseEntity<String> deleteGuardGuideline(@MemberId Long memberId, @RequestParam("guidelineId") Long guidelineId) {
+        guardGuidelineService.deleteGuardGuideline(memberId, guidelineId);
+        return ResponseEntity.ok("가이드라인이 삭제되었습니다.");
+    }
```

Comment:
```suggestion
    @DeleteMapping("/{guidelineId}")
    public ResponseEntity<String> deleteGuardGuideline(@MemberId Long memberId, @PathVariable("guidelineId") Long guidelineId) {
        guardGuidelineService.deleteGuardGuideline(memberId, guidelineId);
        return ResponseEntity.ok("가이드라인이 삭제되었습니다.");
    }
```

좀 더 RESTful 한 느낌이 어떨까 싶어요!~ 한번 제안드려봅니다 😀

Reply:
엇 리뷰에서 이걸 놓쳤네요
성훈님 의견에 동의합니다

Reply:
앗 이 부분 놓쳤네요
delete 사용하는 것으로 수정해두겠습니다:>
혹시 query parameter를 path variable로 바꾸는 것을 제안하시는 이유를 알 수 있을까요?
저는 주소창에서 url을 변경해서 다른 가이드라인을 지우는 것이 적절하지 않을 것 같다고 생각해서 일단은 query parameter로 작성했었습니다

Reply:
은선님 말씀도 맞다고 생각합니다! 저는 큰 의미는 없었고, 저희의 다른 Delete API 들이 path variable로 URL을 만들어 놓은것을 보고 저렇게 제안드려봤어요. 은선님 도메인이시니 뭐든 선택해도 좋으실거 같습니다.

Reply:
넵 수정해두었습니다~!